plugins/CoreUpdater/Updater.php

```diff
@@ -151,6 +182,12 @@ public function oneClickUpdatePartTwo()
     {
         $messages = [];
 
+        if (!Marketplace::isMarketplaceEnabled()) {
```

Comment:
had marketplace disabled locally and it was actually before resulting in below error but because we don't show messages we never noticed basically